docs/release-notes/rockcraft-1-7-0.rst

```diff
@@ -0,0 +1,101 @@
+.. _release-1.7.0:
+
+Rockcraft 1.7.0 release notes
+=============================
+
+20 December 2024
+
+Learn about the new features, changes, and fixes introduced in Rockcraft 1.7.0.
+For information about the Rockcraft release cycle, see the
+:ref:`release_policy_and_schedule`.
+
+
+What's new
+----------
+
+Rockcraft 1.7.0 brings the following features, integrations, and improvements.
+
+
+Support for remote builds
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We added support for remote builds, which leverage off-site compute for
+building rocks and can build for multiple architectures with a single command.
+Remote builds are hosted on Launchpad.
+
+This feature is experimental and subject to change. Feel free to provide
+feedback on the `Ubuntu Discourse`_ forum.
+
+Minor features
+--------------
+
+Rockcraft 1.7.0 brings the following minor changes.
+
+
+"init" command
+~~~~~~~~~~~~~~
+
+- The default base of the project file generated by the "rockcraft init" command has
+  been updated from "ubuntu@22.04" to "ubuntu@24.04".
```

Comment:
maybe this was my confusion: should these be `"ubuntu@22.04"` or inline-code-formatted?

Reply:
One more thought about inline code, while I'm thinking about this:

Sometimes a symbol can become a real word. For example, core22 and commands like grep, which were originally intended as machine expressions, have entered general parlance. Things like that, which have become actual words everyday speech, don't need code fences. Determining whether something is a word can sometimes be difficult, so you can leave it up to me.